python/MeshData.sip

```diff
@@ -17,4 +17,10 @@ public:
     void setVerticesFromBytes(const bytearray& data);
 
     void setFacesFromBytes(const bytearray& data);
-};
\ No newline at end of file
+
+    bytearray getUVCoordinatesPerVertexAsBytes(const Scene* scene) const;
+
+    void setUVCoordinatesPerVertexAsBytes(const bytearray& data, const std::string& texture_path, Scene* scene);
+
+    std::string getTexturePath(const Scene* scene) const;
```

Comment:
there is no way to set a texture path?

Reply:
I don't understand, you set the texture path by calling the `setUVCoordinatesPerVertexAsBytes` function? I would expect to have to function, something akin to `setUV` and `setTextureImage`. You're saying `setUVCoordinatesPerVertexAsBytes` does both?

Reply:
Yes indeed... The UV coordinates are actually stored in two places:
* The coordinates themselves are stored in the global resources
* The mesh stores indices of the coordinates for each of its faces
Then the texture is also stored in a separate resource, linked from the coordinates. So in order to simplify the management and avoid mistakes, I decided to add it all to a single call. But maybe I should rename the function then, because it does more than advertised.